components/prize_pool/commons/prize_pool.lua

```diff
@@ -1026,15 +1038,16 @@ function Placement:_getLpdbData()
 
 		local prizeMoney = tonumber(self:getPrizeRewardForOpponent(opponent, PRIZE_TYPE_USD .. 1)) or 0
 		local pointsReward = self:getPrizeRewardForOpponent(opponent, PRIZE_TYPE_POINTS .. 1)
+		local opponentType = opponent.opponentData.type
 		local lpdbData = {
 			image = image,
 			imagedark = imageDark,
 			date = opponent.date,
 			participant = participant,
 			participantlink = Opponent.toName(opponent.opponentData),
-			participantflag = (players or {}).p1flag,
+			participantflag = opponentType == Opponent.solo and (players or {}).p1flag or nil,
```

Comment:
```suggestion
			participantflag = opponentType == Opponent.solo and players.p1flag or nil,
```
If opponent is solo then player is guaranteed to exist.

Reply:
true